.github/workflows/check.yaml

```diff
@@ -75,6 +75,7 @@ jobs:
         with:
           preview: 'true'
           token: ${{ secrets.REPO_GHA_PAT }}
+          extra-params: '-H "${{ secrets.SWARM_CLI_EXTRA_FLAGS }}"'
```

Comment:
Wouldn't be better to call this `SWARM_CLI_EXTRA_HEADER`? Or even say `GATEWAY_AUTHORIZATION_HEADER`?